dist/images/ovndb-raft-functions.sh

```diff
@@ -217,7 +216,7 @@ ovsdb-raft() {
     fi
     run_as_ovs_user_if_needed \
       ${OVNCTL_PATH} run_${db}_ovsdb --no-monitor \
-      --db-${db}-cluster-local-addr=${local_ip} --db-${db}-cluster-remote-addr=${init_ip} \
+      --db-${db}-cluster-local-addr=${ovn_db_host} --db-${db}-cluster-remote-addr=${init_ip} \
```

Comment:
`ovn-ctl` requires that the arguments to `--db-{nb,sb}-cluster-{local,remote}-addr` have brackets around them if they're IPv6 addresses. (eg, `--db-nb-cluster-local-addr=[fd01::1234]`)

In OpenShift's scripts we define a helper function

          bracketify() { case "$1" in *:*) echo "[$1]" ;; *) echo "$1" ;; esac }

and then do, eg

            --db-nb-cluster-local-addr=$(bracketify ${K8S_NODE_IP}) \
            --db-nb-cluster-remote-addr=$(bracketify ${MASTER_IP}) \

Reply:
@danwinship we have a separate PR to add IPv6 support to `ovnkube.sh`. We will address this in that PR. Thanks.